packages/babel-plugin-transform-es2015-classes/test/fixtures/loose/accessing-super-class/expected.js

```diff
@@ -1,20 +1,18 @@
 var Test = function (_Foo) {
-  babelHelpers.inherits(Test, _Foo);
+  babelHelpers.inheritsLoose(Test, _Foo);
 
   function Test() {
     var _Foo$prototype$test, _Foo$prototype$test2;
 
-    babelHelpers.classCallCheck(this, Test);
-
     woops.super.test();
 
-    var _this = babelHelpers.possibleConstructorReturn(this, _Foo.call(this));
+    var _this = _Foo.call(this) || this;
```

Comment:
If we do, we'd have to remove the whole `var _this = ` (and all the `_this`s) too. I think this is fine for now.